lms/envs/common.py

```diff
@@ -1244,6 +1244,13 @@
     'js/vendor/url.min.js',
     'common/js/vendor/underscore.js',
     'js/vendor/underscore.string.min.js',
+
+    # Make some edX UI Toolkit utilities available in the global "edx" namespace
+    'edx-ui-toolkit/js/utils/global-loader.js',
```

Comment:
I'm OK with it, though I realize that we keep adding more bloat to base_vendor (which impacts performance).